bindings/electron/src/index.d.ts

```diff
@@ -959,3 +959,39 @@ export function testGetTestbedBootstrapOrganizationAddr(
 export function testDropTestbed(
     path: string
 ): Promise<null>
+export function validateEntryName(
+    raw: string
+): Promise<boolean>
+export function validatePath(
+    raw: string
+): Promise<boolean>
+export function validateHumanHandleLabel(
+    raw: string
+): Promise<boolean>
+export function validateEmail(
+    raw: string
+): Promise<boolean>
+export function validateDeviceLabel(
+    raw: string
+): Promise<boolean>
+export function validateInvitationToken(
+    raw: string
+): Promise<boolean>
+export function validateBackendAddr(
```

Comment:
Backend addresses can already be validated using `parseBackendAddr` (with a bit more precision even, being able to tell between UserInvitation and DeviceInvitation), so there's not really any need for these functions. The frontend already uses `parseBackendAddr`, and this just adds code that will not be used and will have to be maintained.

Reply:
The functions that check `BackendAddr*` have been removed